src/server/views/help/privacy-notice.html

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.html" %}
+
+{% set pageTitle = "Submit a form to Defra privacy notice" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Submit a form to Defra privacy notice</h1>
+      <p class="govuk-body">The {{ form.title }} form was created using ‘Submit a form to Defra’. This service is owned and operated by the Department for Environment, Food & Rural Affairs (Defra).</p>
+
+      <h2 class="govuk-heading-m">Who collects your personal data</h2>
+      <p class="govuk-body">The organisation that created the {{ form.title }} form using ‘Submit a form to Defra’ is the data controller of personal data they collect. If the data controller is outside the Defra legal entity, then Defra is the data processor.</p>
+      <p class="govuk-body">Read the <a href="{{ form.privacyNoticeUrl }}" class="govuk-link">specific privacy notice for the {{ form.title }} form</a>.</p>
+
+      <p class="govuk-body">Defra also collects some data as a data controller. This privacy notice explains what personal data Defra collects and processes as a data controller through forms made with ‘Submit a form to Defra’.</p>
+      <p class="govuk-body">If you need further information about how Defra uses your personal data, email <a href="mailto:defraforms@defra.gov.uk" class="govuk-link">defraforms@defra.gov.uk</a>.</p>
+      <p class="govuk-body">If you want information and your associated rights you can email: <a href="mailto:data.protection@defra.gov.uk" class="govuk-link">data.protection@defra.gov.uk</a>.</p>
+      <p class="govuk-body">The data protection officer for Defra is responsible for checking that Defra complies with legislation. You can contact them at <a href="mailto:DefraGroupDataProtectionOfficer@defra.gov.uk" class="govuk-link">DefraGroupDataProtectionOfficer@defra.gov.uk</a>.</p>
```

Comment:
On smaller screens, we might have to watch out for long emails either now or when the [new type scale](https://design-system.service.gov.uk/get-started/new-type-scale/) lands. Rather than wrap, they'll cause the page to have horizontal scroll bars

Can either [force breaks with `govuk-!-text-break-word`](https://design-system.service.gov.uk/styles/font-override-classes/#breaking-up-long-words) or [add line break opportunities via `<wbr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/wbr)

```suggestion
      <p class="govuk-body">The data protection officer for Defra is responsible for checking that Defra complies with legislation. You can contact them at <a href="mailto:DefraGroupDataProtectionOfficer@defra.gov.uk" class="govuk-link">Defra<wbr>Group<wbr>Data<wbr>Protection<wbr>Officer@defra.gov.uk</a>.</p>
```

Reply:
Resolved with b9853d22b41fd9b4474186da05ed1a7d5a2d2658